src/Types/Scripts.purs

```diff
@@ -41,22 +41,22 @@ derive instance Generic PlutusScript _
 derive instance Newtype PlutusScript _
 derive newtype instance Eq PlutusScript
 derive newtype instance Ord PlutusScript
-derive newtype instance DecodeJson PlutusScript
-derive newtype instance EncodeJson PlutusScript
+derive newtype instance DecodeAeson PlutusScript
+derive newtype instance EncodeAeson PlutusScript
 
 instance Show PlutusScript where
   show = genericShow
 
-decodeJsonHelper
+decodeAesonHelper
   ∷ ∀ (a ∷ Type) (b :: Type)
```

Comment:
This is not you but are we okay with unicode? We should probably stick to one since we're generally trying to standardise our code @ngua

Reply:
There's also usage in `Cardano.Types.Value.union`

Reply:
I think this is lsp being smart :/

Reply:
Oh, I would prefer not to use unicode. Not sure where that was introduced